manifests/init.pp

```diff
@@ -104,6 +104,8 @@
 #
 # $https_chain::                apache chain file for ssl
 #
+# $https_includes::             additional includes for https vhost
```

Comment:
I'd be good to note that this only applies when ```manage_httpd``` is true.

Reply:
Oh hey, thanks @ekohl! I did completely miss that, that will work just fine.  You can close this out, unless you have some other use for it (seems doubtful)